ui/src/plugins/dev.perfetto.ExplorePage/styles.scss

```diff
@@ -133,3 +133,72 @@
     margin-bottom: 16px;
   }
 }
+
+.query-canvas-container {
```

Comment:
query-node-container? I think canvas and container are two words for the same thing here